tasks/alignment/task_bwa.wdl

```diff
@@ -52,14 +127,18 @@ task bwa {
     File sorted_bai = "${samplename}.sorted.bam.bai"
     File read1_aligned = "~{samplename}_R1.fastq.gz"
     File? read2_aligned = "~{samplename}_R2.fastq.gz"
+    File read1_unaligned = "~{samplename}_unaligned_R1.fastq.gz"
+    File? read2_unaligned = "~{samplename}_unaligned_R2.fastq.gz"
+    File sorted_bam_unaligned = "~{samplename}.sorted.unaligned-reads.bam"
```

Comment:
I'm having an existential crisis on if we should rename `sorted_bam` and `sorted_bai` as `sorted_aligned_bam` and `sorted_aligned_bai` for readability :sweat_smile: 
Additionally, I would rename `sorted_bam_unaligned` to `sorted_unaligned_bam` so the variable has the file type at the end. It's a bit cleaner and it goes with or style-guide. Same for `sorted_bam_unaligned_bai` -> `sorted_unaligned_bai`

Reply:
Personally I prefer names that tell a story of what has been done to the file, in which case the bam file was sorted, and then unaligned reads extracted. Happy to hear what others think and this could feed onto the style-guide.